GiniVision/Classes/Networking/Extensions/GiniScreenAPICoordinator+Networking.swift

```diff
@@ -203,6 +203,20 @@ extension GiniScreenAPICoordinator: GiniVisionDelegate {
     }
     
     func didCapture(document: GiniVisionDocument, networkDelegate: GiniVisionNetworkDelegate) {
+        // The EPS QR codes are a special case, since they cannot be analyzed by the Gini API and therefore, they are
```

Comment:
I wouldn't say they "cannot" but "they don't have to be"

Reply:
Better, yeah